Source/WebKit/Configurations/BaseXPCService.xcconfig

```diff
@@ -37,6 +37,7 @@ APP_ICON_Release = WebKit.icns;
 APP_ICON_Production = ;
 
 EXCLUDED_SOURCE_FILE_NAMES[sdk=embedded*] = $(EXCLUDED_SOURCE_FILE_NAMES_$(CONFIGURATION)) *.xib MediaFormatReader.bundle;
+EXCLUDED_SOURCE_FILE_NAMES_YES[sdk=iphone*17.4*] = *;
```

Comment:
I believe this excludes all of the XPC build files for iPhoneOS and iPhoneSimulator....